roles/kubernetes/preinstall/tasks/0080-system-configurations.yml

```diff
@@ -22,6 +23,26 @@
   tags:
     - bootstrap_os
 
+# SELinux might still be disabled on this host, pending reboot
+- name: Get SELinux state
+  ansible.builtin.setup:
+    gather_subset: selinux
```

Comment:
Can this be folded into the setup call earlier in the playbooks (from internal_facts playbook or bootstrap-os role) ?